openhabian.conf.dist

```diff
@@ -3,7 +3,9 @@
 # This file will only be used on initial install.
 # If you want to change anything after install completed, use menu in openhabian-config.
 
+configfile=/etc/openhabian.conf
 hostname=openHABianDevice
+username=openhabian
```

Comment:
may not be needed, the dist fle is only used for manual install. not sure if then a openhabian user is created. but we can sort out this later.

Reply:
ok will leave it (it was inconsistent)